DOM_creation/lesson1/index.html

```diff
@@ -1,5 +1,6 @@
+<!-- Lesson1 -->
```

Comment:
nits
please remove this because your already know by folder name

Reply:
Thank you for your comment Morita-san!

As per your feedback, deleted the comment line(s).
Kindly check the latest commit 
5c9ba1f9a6e5a93bc0d93f44d1688e1bfbe43b68